### PR TITLE
Fix FPR and RPR for 2p GasOil case

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

File: opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -375,11 +375,16 @@ quantity fpr( const fn_args& args ) {
 
     const auto& p = args.state.data( "PRESSURE" );
     const auto& pv = args.pv;
-    const auto& sw = args.state.data( "SWAT" );
+    const bool hasSwat = args.state.has( "SWAT" );
+    const auto& swat = hasSwat? args.state.data( "SWAT" ): p;
```
Comment (joakim-hove) [inline opm/output/eclipse/Summary.cpp]: No; you have to rewrite this (eg using a pointer?); initializing the variable `swat` to pressure if the model does not have `SWAT`is just to error prone.

Comment (totto82) [reply opm/output/eclipse/Summary.cpp]: Is this better?

**Comment by joakim-hove** [comment]
Personally I would do something like: 

```C++
if (args.state.has("SWAT")) {
    const auto& swat = args.state.data("SWAT");
    for (size_t i = 0; i < p.size(); i++)
          hcs += pv[i] * (1 - swat[i])
} else {
    for (size_t i; i < p.size(); i++)
         hcs += pv[i];
}
```
but if you prefer your variety I will merge that.

What about the test failure downstream?

**Comment by totto82** [comment]
> What about the test failure downstream?

Strange SPE5 doesn't even output FPR.

**Comment by totto82** [comment]
SPE5 fails for everybody. 
I guess we should blame: OPM/ewoms#219

**Comment by GitPaean** [comment]
What happened with Jenkins when testing with that PR?

**Comment by atgeirr** [comment]
jenkins build this with downstreams please

**Pull request merged**